src/manifest/mod.rs

```diff
@@ -169,7 +142,46 @@ impl CargoManifest {
             files: files,
             main: js_file,
             types: dts_file,
+        })
+    }
+
+    fn into_es6(mut self, scope: &Option<String>, disable_dts: bool) -> NpmPackage {
+        let filename = self.package.name.replace("-", "_");
+        let wasm_file = format!("{}_bg.wasm", filename);
+        let js_file = format!("{}.js", filename);
+
+        let dts_file = if disable_dts == true {
+            None
+        } else {
+            Some(format!("{}.d.ts", filename))
+        };
+
+        if let Some(s) = scope {
+            self.package.name = format!("@{}/{}", s, self.package.name);
         }
+        let mut files = vec![wasm_file];
+
+        match dts_file {
+            Some(ref dts_file) => {
+                files.push(dts_file.to_string());
+            }
+            None => {}
+        }
+
+        NpmPackage::ES6Package(ES6Package {
```

Comment:
nitpick: I'd probably name this `EsModulePackage` since I think modules didn't come until after es6 and es6 is also actually es2015 or some such... or something like that, right?

Reply:
yeah that actually makes a bunch of sense, good catch